src/threadctx.hpp

```diff
@@ -52,6 +52,11 @@ struct ThreadCtx {
 
     MatDynMem response{roi, CV_32FC(int(num_of_scales))};
 
+    KCF_Tracker::GaussianCorrelation* get_gaussian_correlation()
```

Comment:
No need to add anything, moving gaussian_correlation from private to public will work.